keras_cv/models/backbones/csp_darknet/csp_darknet_backbone_presets.py

```diff
@@ -0,0 +1,144 @@
+# Copyright 2023 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""CSPDarkNet model preset configurations."""
+
+backbone_presets_no_weights = {
+    "cspdarknettiny": {
```

Comment:
Use underscore separators to match other presets and improve readability:
```
"csp_dark_net_tiny",
"csp_dark_net_s",
...
```

Reply:
Done.